README.md

```diff
@@ -144,12 +144,20 @@ import UserNotifications
 ...
 
 let center = UNUserNotificationCenter.current()
-center.delegate = self as? UNUserNotificationCenterDelegate
-let options: UNAuthorizationOptions = [.alert, .sound, .badge, .provisional]
-
-center.requestAuthorization(options: options) { (granted, error) in
-    // Enable / disable features based on response
+center.delegate = self as? UNUserNotificationCenterDelegate // the type casting can be removed once the delegate has been implemented
+let options: UNAuthorizationOptions = [.alert, .sound, .badge]
+// use the below options if you are interested in using provisional push notifications. Note that using this will not
+// show the push notifications prompt to the user.
+// let options: UNAuthorizationOptions = [.alert, .sound, .badge, provisional]
```

Comment:
Do you want this commented out?

Reply:
Yeah since line 148 has the `options` without `. provisional`

Reply:
I might reverse them. I think we should push them towards provisional.

Reply:
I agree that provisional notifications is better for users to start engaging with their clients but my only concern with that is for folks trying to setup regular push without provisional notifications might get confused when they don't see the prompt to the user to accept or decline push. Also, since we don't have any support articles around this it might be better to have it as a feature that's there if you want it but we don't want to present it as the default behavior until we fully support it. What'd you think?

Reply:
I agree with Ajay in the absence of a more robust support article I'd default to not showing .provisional in the example... even though I also agree its the option we should probably encourage people to use.